velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -30,118 +30,96 @@ static const std::vector<std::string> kAggregateFunctions = {
     std::string("avg(c2)"),
     std::string("sum(1)")};
 
-struct TestParam {
-  const std::string aggregateFunction;
-  const std::string frameClause;
+// This WindowFunctionTest class is used to instantiate parameterized
+// window function tests. The parameters are based on the function being
+// tested and a specific over clause. The window function is tested for
+// the over clause and all combinations of frame clauses.
+struct WindowTestParam {
+  const std::string function;
+  const std::string overClause;
 };
 
-std::vector<TestParam> getTestParams(
-    const std::vector<std::string>& frameClauses) {
-  std::vector<TestParam> params;
-  for (auto aggregateFunction : kAggregateFunctions) {
-    for (auto frameClause : frameClauses) {
-      params.push_back({aggregateFunction, frameClause});
-    }
-  }
-  return params;
-}
-
-class SimpleAggregatesTest : public WindowTestBase {
+class WindowFunctionTest : public WindowTestBase {
```

Comment:
Is this a base class for other test classes? If so, let's rename to XxxTestBase.

Reply:
This isn't a base class for other test classes. Have modified the code to look more relevant now.